resources/integration_test/testdata/integration.yml

```
@@ -18,3 +18,25 @@ dataset:
       - {_id: 5, studentid: 10329, testid: 5,  score: 74.4}
       - {_id: 6, studentid: 342, testid: 5,  score: 80.1}
       - {_id: 7, studentid: 204323, testid: 5, score: 83.3}
+
+  - db: integration_test
+    collection: types_other
+    docsJson:
+      - {
+        _id: 0,
+        dbPointer: { "$dbPointer": { "$ref": "namespace", "$id": { "$oid": "000000000000000000000001" } } },
+        javascript: {"$code": "function(){ }"},
+        javascriptWithScope: {"$code": "function(){ }", "$scope": {"foo":"bar"}},
+        maxKey: {"$maxKey": 1},
+        minKey: {"$minKey": 1},
+        object: {"foo":"bar"},
+        objectId: { "$oid": "000000000000000000000001" },
+        regularExpression: {"$regularExpression": {pattern: "a(bc)*", "options": ""}},
+        symbol: { "$symbol": "symbol" },
+        timestamp: {"$timestamp": {"t":100, "i":0}}
+
+        # Skip reason: SQL-395
+        # undefined: {"$undefined":true},
+        # Skip reason: SQL-697
+        # array: [1,2,3]
+      }
```

```suggestion
      }

```
[nit] newlines at end of files